src/resolvers/delta.resolver.ts

```diff
@@ -0,0 +1,28 @@
+import {OptionsResolver} from '../options-resolver'
+import {Resolver} from './resolver'
+import {Password} from 'symbol-sdk'
+import {Options} from 'clime'
+import {DeltaValidator} from '../validators/delta.validator'
+
+/**
+ * Password resolver
+ */
+export class DeltaResolver implements Resolver {
+
+    /**
+     * Resolves a delta value provided by the user.
+     * @param {Options} options - Command options.
+     * @param {string} altText - Alternative text.
+     * @param {string} altKey - Alternative key.
+     * @returns {Promise<Password>}
+     */
+    async resolve(options: Options, altText: string, altKey?: string): Promise<number> {
+        const resolution = await OptionsResolver(options,
+            altKey ? altKey : 'delta',
+            () => undefined,
+            altText,
+            'number',
+            new DeltaValidator())
+        return +resolution
```

Comment:
not sure whether there should be a + here

Reply:
This is the same as writting ``Number(resolution)``. The ``OptionsResolver`` response is not typed, so if we don't parse it as a number the compiler complains.

Reply:
nice, I didn't know this trick!